src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ContextQuery/SyntaxNodeExtensions.cs

```diff
@@ -30,5 +30,7 @@ public static bool IsDelegateOrConstructorOrLocalFunctionOrMethodOrOperatorParam
 
             return false;
         }
+
+        public static bool IsInFunctionPointer(this SyntaxToken node) => node.Parent.IsKind(SyntaxKind.FunctionPointerType);
```

Comment:
this isn't named well (And i don't think we need this extension at all).  "is in" implies to me it's in the subtree of that function-pointer, not that its' directly parented by the function pointer.  rename, or just inline this at callers.

Reply:
Removed.